eli5/base.py

```diff
@@ -25,6 +25,8 @@ def __init__(self,
                  decision_tree=None,  # type: Optional[TreeInfo]
                  highlight_spaces=None,  # type: Optional[bool]
                  transition_features=None,  # type: Optional[TransitionFeatureWeights]
+                 image=None, # type: object
```

Comment:
If this can be anything, it's better to use ``Any`` type, but for image and heatmap it seems it's possible to be more specific? E.g. ``Optional[np.ndarray]``?

Reply:
Good catch. I will annotate heatmap with `Optional[np.ndarray]` and keep image as `Any` (don't want to import `PIL` as it is optional).

In the future we can start using the solution presented at https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles, where we place imports needed only for annotations under a `if TYPE_CHECKING:` statement.

Reply:
I see, this makes sense, thank you!